k4FWCore/include/k4FWCore/DataWrapper.h

```diff
@@ -50,7 +49,7 @@ template <class T> class GAUDI_API DataWrapper : public DataWrapperBase {
     uptr.release();
     is_owner = false;
   }
-  virtual ~DataWrapper();
+  virtual ~DataWrapper() override;
```

Comment:
```suggestion
  ~DataWrapper() override;
```
`virtual` is redundant with `override`

Reply:
Yeah, well depending on who you ask `virtual` happens before and it's clearer to show intention :shrug: